extension/src/popup/views/Account/index.tsx

```diff
@@ -206,15 +213,26 @@ export const Account = () => {
                   </div>
                 </div>
               </div>
-              {isFunded ? (
+              {hasError && (
```

Comment:
I split these out to avoid showing an NotFundedMessage when we fail to fetch account details.